src/corecel/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ set(CELERITAS_USE_HEPMC3  ${CELERITAS_USE_HepMC3})
 set(CELERITAS_USE_OPENMP  ${CELERITAS_USE_OpenMP})
 set(CELERITAS_USE_VECGEOM ${CELERITAS_USE_VecGeom})
 set(CELERITAS_USE_PERFETTO ${CELERITAS_USE_Perfetto})
+set(CELERITAS_USE_COVFIE ${CELERITAS_USE_Covfie})
```

Comment:
Please also add the covfie version on line 97, and in the `corecel/io/BuildOutput.cc`. Also let's keep this list sorted.